app/[locale]/projects/[projectId]/page.tsx

```diff
@@ -314,7 +322,10 @@ const ProjectOverviewPage: React.FC = () => {
                     </div>
                     <div className="plan-action">
                       <Link
-                        href={`/projects/${projectId}/dmp/xxx`}
+                        href={routePath('projects.dmp.show', {
+                          projectId: String(projectId),
+                          dmpId: planId
+                        })}
```

Comment:
I think @jupiter007 also made this fix in a PR recently. Make sure to re-merge dev and resolve conflicts into your branch just to be sure.